prac_09/unreliable_car_test.py

```diff
@@ -0,0 +1,13 @@
+"""Test UnreliableCar"""
+
+from prac_09.unreliable_car import UnreliableCar
+
+
+def main():
```

Comment:
This test code could be improved by using assert statements or by comparing the "Expected" and "Got" results.

Reply:
I didn't include them because the instructions didn't specify, but I agree, especially since the reliability aspect would benefit from more iterations

Reply:
I didn’t include those because the instructions didn’t specify and there is randomness involved, but I might implement more rigorous testing since the new functionality would benefit from more iterations